granian/_loops.py

```diff
@@ -8,9 +8,9 @@
 
 
 class Registry:
-    __slots__ = ['_data']
+    __slots__ = ('_data',)
 
-    def __init__(self):
+    def __init__(self) -> None:
```

Comment:
Isn't `-> None` implicit? Do we really need to bloat code with such useless annotations?